eureka-client/src/main/java/com/netflix/discovery/shared/transport/jersey/JerseyEurekaHttpClientFactory.java

```diff
@@ -153,7 +162,7 @@ private JerseyEurekaHttpClientFactory buildLegacy() {
                     .withReadTimeout(readTimeout)
                     .withMaxConnectionsPerHost(maxConnectionsPerHost)
                     .withMaxTotalConnections(maxTotalConnections)
-                    .withConnectionIdleTimeout(connectionIdleTimeout)
+                    .withConnectionIdleTimeout((int) connectionIdleTimeout)
```

Comment:
If this is the majority use case and inside EurekaJerseyClientImpl we use the idle timeout as an int, why use long in the first place?